From 3c24beb9cff8310b3ba9bd202ebaf24915754423 Mon Sep 17 00:00:00 2001
From: Anupam Bhatnagar <anupamb@fb.com>
Date: Wed, 9 Mar 2022 19:11:43 +0000
Subject: [PATCH 1/2] 0.4.6 release

---
 fairscale/version.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fairscale/version.py b/fairscale/version.py
index c11e915..130c0e4 100644
--- a/fairscale/version.py
+++ b/fairscale/version.py
@@ -1 +1 @@
-__version_tuple__ = (0, 4, 5)
+__version_tuple__ = (0, 4, 6)
-- 
2.17.1

